demosplan/DemosPlanCoreBundle/Controller/User/DemosPlanOrganisationAPIController.php

```diff
@@ -321,18 +325,26 @@ public function createOrgaAction(Request $request,
             }
 
             // Add new permission in case it is present in the request
-
+            $canCreateProcedures = null;
             if ($permissions->hasPermission('feature_manage_procedure_creation_permission')
                 && array_key_exists('canCreateProcedures', $orgaDataArray)) {
                 $roles = $roleHandler->getUserRolesByCodes([RoleInterface::PRIVATE_PLANNING_AGENCY]);
                 Assert::count($roles, 1);
```

Comment:
Has nothing really to do with this PR but as a side effect: Could you perhaps include the try/catch here as well?

Reply:
@mussbach done 2b427400e9f616036b226cc6c46b410f54188df7